runtime/syntax/sh.yaml

```
@@ -39,7 +39,7 @@ rules:
       # Coreutils commands
     - type: "\\b(base64|basename|cat|chcon|chgrp|chmod|chown|chroot|cksum|comm|cp|csplit|cut|date|dd|df|dir|dircolors|dirname|du|env|expand|expr|factor|false|fmt|fold|head|hostid|id|install|join|link|ln|logname|ls|md5sum|mkdir|mkfifo|mknod|mktemp|mv|nice|nl|nohup|nproc|numfmt|od|paste|pathchk|pinky|pr|printenv|printf|ptx|pwd|readlink|realpath|rm|rmdir|runcon|seq|(sha1|sha224|sha256|sha384|sha512)sum|shred|shuf|sleep|sort|split|stat|stdbuf|stty|sum|sync|tac|tail|tee|test|time|timeout|touch|tr|true|truncate|tsort|tty|uname|unexpand|uniq|unlink|users|vdir|wc|who|whoami|yes)\\b"
       # Conditional flags
-    - statement: "\\s+(-[A-Za-z]+|--[a-z]+)"
+    - statement: "\\s+(-[A-Za-z]+|--[a-z-]+)"
```

To pickup the improvements mentioned by @Andriamanitra:

```suggestion
    - statement: "\\s+(-[0-9A-Za-z]+|--[0-9A-Za-z_-]+)"
```

Additionally we should add single-hyphen options with numbers, as used by `rsync` with `-4` or `-6`.